client/app/components/partials/DefineAxisModal.vue

```diff
@@ -1,172 +1,165 @@
-                        <input id="yMin" v-model="yMin" :placeholder="yMinPlaceholderText">
-                      </div><!-- /form-group -->
-                      <div class="form-group">
-                        <label class="form-control-label" for="yMax">yMax</label>
-                        <input id="yMax" v-model="yMax" :placeholder="yMaxPlaceholderText">
-                      </div><!-- /form-group -->
-                    </form>
-                  </slot>
-                </div>
-
-                <div class="modal-footer">
-                  <slot name="footer">
-                    <button class="btn btn-default" @click.prevent="reset">
-                      Reset
-                    </button>
-                    <button class="btn btn-default" @click.prevent="save">
-                      Save
-                    </button>
-                  </slot>
-                </div>
-
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </transition>
-
+	<transition name="modal">
+		<div class="modal" style="display: block" @click="$emit('close')">
+			<div class="modal-mask">
+				<div class="modal-wrapper" @click.stop>
+					<div class="modal-dialog">
+						<div class="modal-content">
+							<div class="modal-header">
+								<slot name="header">
+									<button type="button" class="close" @click="$emit('close')">&times;</button>
+									<h4 class="modal-title">Define custom ranges for axes:</h4>
+								</slot>
+							</div>
+
+							<div class="modal-body">
+								<slot name="body">
+									<form>
+										<div class="form-group">
+											<label class="form-control-label" for="xMin">xMin</label>
+											<input id="xMin" v-model="xMin" :placeholder="xMinPlaceholderText" />
+										</div>
+										<!-- /form-group -->
+										<div class="form-group">
+											<label class="form-control-label" for="xMax">xMax</label>
+											<input id="xMax" v-model="xMax" :placeholder="xMaxPlaceholderText" />
+										</div>
+										<!-- /form-group -->
+										<div class="form-group">
+											<label class="form-control-label" for="yMin">yMin</label>
+											<input id="yMin" v-model="yMin" :placeholder="yMinPlaceholderText" />
+										</div>
+										<!-- /form-group -->
+										<div class="form-group">
+											<label class="form-control-label" for="yMax">yMax</label>
+											<input id="yMax" v-model="yMax" :placeholder="yMaxPlaceholderText" />
+										</div>
+										<!-- /form-group -->
+									</form>
+								</slot>
+							</div>
+
+							<div class="modal-footer">
+								<slot name="footer">
+									<button class="btn btn-default" @click.prevent="reset">Reset</button>
+									<button class="btn btn-default" @click.prevent="save">Save</button>
+								</slot>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</transition>
 </template>
 
 <script>
 export default {
```

Comment:
this file didn't have an empty line, odd... still, let's add it for consistency
"leave things better than we found them" and all that
```suggestion

export default {
```